### PR TITLE
Performance benchmark

### PR DESCRIPTION
**Pull request by babenek**
## Description

- added performance test against released version
- ``--banner`` might be passed as single argument

## How has this been tested?

- [x] benchmark workflow
- [x] 10% - significant level

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -146,3 +146,113 @@ jobs:
...
+
+    - name: Update PIP
+      run: python -m pip install --upgrade pip
+
+    - name: Install released CredSweeper
+      run: python -m pip install credsweeper
+
+    - name: Run performance benchmark
+      run: |
+        credsweeper --path requirements.txt --banner
```
Comment (csh519) [inline .github/workflows/benchmark.yml:195]: Why is it needed? `requirements.txt` file scanning?

Comment (babenek) [reply .github/workflows/benchmark.yml:195]: It's time to do --banner self-sufficient!

Comment (csh519) [reply .github/workflows/benchmark.yml]: Thank you for fix!

**Review by csh519 — APPROVED** [review]
Checking performance of benchmark data is needed.
And adding `--banner` option as a standalone also good.

LGTM 👍

**Review by kmnls — APPROVED** [review]

**Pull request merged**